src/common/libjob/job.h

```diff
@@ -51,6 +51,14 @@ typedef enum {
     FLUX_JOB_INACTIVE               = 32,   // captive end state
```

Comment:
English in comment:

> Updating the flux-info.list protocol so that the list flags
can be dropped will be for another PR against issue #2628.

"will be for" -> "in"?

Reply:
Yeah, sorry awkward sentence, and probably something more appropriate to communicate in the PR rather than a commit.  I'll add #2628 to the mention in the earlier sentence.